packages/contentstack-import/src/import/modules-js/custom-roles.js

```diff
@@ -110,6 +110,15 @@ module.exports = class ImportCustomRoles {
       log(self.config, chalk.green('Custom-roles have been imported successfully!'), 'success');
     } catch (error) {
       fileHelper.writeFileSync(customRolesFailsPath, self.fails);
+              let message_content_type = '';
+        if (error.request !== undefined && JSON.parse(error.request.data).role !== undefined) {
+          if (JSON.parse(error.request.data).role.name) {
+            message_content_type =
+              ' Due to custom-role named  - ' + JSON.parse(error.request.data).role.name;
```

Comment:
@cs-raj Can you please validate the error messages with the docs team?

Reply:
Validated with Varsha and made the changes